Yosemite/Yosemite/Stores/StatsV4AvailabilityStore.swift

```diff
@@ -0,0 +1,45 @@
+import Networking
+
+class StatsV4AvailabilityStore: Store {
```

Comment:
I think this store can be abstracted to a wider `AvailabilityStore` that potentially we can expand to not only check the availability of stats, but other remote services like plugins or certain modules (i.e. stats) being disabled, so I would suggest renaming it to `AvailabilityStore`.

Also, it should probably be final, unless there is a good reason to potentially subclass it later on?

Reply:
> I think this store can be abstracted to a wider AvailabilityStore that potentially we can expand to not only check the availability of stats, but other remote services like plugins or certain modules (i.e. stats) being disabled, so I would suggest renaming it to AvailabilityStore.

yea this makes sense! will update

> Also, it should probably be final, unless there is a good reason to potentially subclass it later on?

yep will update the class to be final (will make it a default unless there is an immediate plan to subclass)